libportal/background.c

```diff
@@ -266,7 +266,9 @@ xdp_portal_request_background (XdpPortal *portal,
  * @result: a [iface@Gio.AsyncResult]
  * @error: return location for an error
  *
- * Finishes the request, and returns `TRUE` if successful.
+ * Finishes the request.
+ *
+ * Returns `TRUE` if successful.
```

Comment:
seems silly to have this right above the `Returns:` line that has the same info

Reply:
Because of how it is displayed it makes sense, see https://libportal.org/method.Portal.request_background_finish.html, the Returns: appear right at the end.

The wording can be improved indeed, but my intention was to make the most of the "current" docs not to write new ones,

Reply:
alright